swarm/storage/mock/db/db.go

```diff
@@ -40,8 +41,8 @@ type GlobalStore struct {
 }
 
 // NewGlobalStore creates a new instance of GlobalStore.
-func NewGlobalStore(path string) (s *GlobalStore, err error) {
-	db, err := leveldb.OpenFile(path, nil)
+func NewGlobalStore() (s *GlobalStore, err error) {
```

Comment:
This is not right at all. This change makes `mock/db` completely useless compared to `mock/mem`. If in-memory mock store is needed there is `mock/mem`. Please leave this package as it is and where in-memory is needed for global store use `mock/mem`. They both implement `mock. GlobalStorer ` interface.